src/jdk.jdwp.agent/share/native/libjdwp/error_messages.c

```diff
@@ -232,6 +232,7 @@ eventText(int i)
         CASE_RETURN_TEXT(EI_THREAD_START)
         CASE_RETURN_TEXT(EI_THREAD_END)
         CASE_RETURN_TEXT(EI_CLASS_PREPARE)
+        CASE_RETURN_TEXT(EI_CLASS_UNLOAD)
         CASE_RETURN_TEXT(EI_CLASS_LOAD)
```

Comment:
Nit: Would it be a little better to place it after EI_CLASS_UNLOAD?

Reply:
If you mean put EI_CLASS_UNLOAD after EI_CLASS_LOAD, I tried to keep the current ordering. If you look in util.h (bottom of this page), it used to have:
```
        EI_CLASS_PREPARE        =  7,
        EI_GC_FINISH            =  8,
        EI_CLASS_LOAD           =  9,
```
and now it has:
``` 
        EI_CLASS_PREPARE        =  7,
        EI_CLASS_UNLOAD         =  8,
        EI_CLASS_LOAD           =  9,
```
There are a couple of other places where this ordering is preserved and I tried to be consistent.

Reply:
Okay, thanks.
It makes sense then.